packages/fxa-content-server/app/styles/_breakpoints.scss

```diff
@@ -40,11 +40,11 @@ $breakpoints: (
 @mixin min-width($breakpoint) {
   @include breakpoint-error-check($breakpoint) {
     @if $breakpoint == 'tablet' {
-      @media (min-width: #{map-get($breakpoints, $breakpoint) + 0px}) and (min-height: (#{map-get($breakpoints, 'mobileLandscape') + 1px})) {
+      @media (min-width: #{map-get($breakpoints, $breakpoint) + 0}) and (min-height: (#{map-get($breakpoints, 'mobileLandscape') + 1px})) {
         @content;
       }
     } @else {
-      @media (min-width: #{map-get($breakpoints, $breakpoint) + 0px}) {
+      @media (min-width: #{map-get($breakpoints, $breakpoint) + 0}) {
```

Comment:
Hey @byrw, thanks for your PR! It actually looks great besides this line, and a similar edit on line 43. Typically we can drop the `px` on `0` CSS values, but this is a weird case where we coerce the value into `px` here so it's required for these mixins. If the linter is complaining, a comment above the line to ignore the next line would be great.

Reply:
Thank you @LZoog.